brazilian-journal-of-experimental-design-data-analysis-and-inferential-statistics.csl

```diff
@@ -0,0 +1,492 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Brazilian Journal of Experimental Design, Data Analysis and Inferential Statistics</title>
+    <title-short>BJEDIS</title-short>
+    <id>http://www.zotero.org/styles/brazilian-journal-of-experimental-design-data-analysis-and-inferential-statistics</id>
+    <link href="http://www.zotero.org/styles/brazilian-journal-of-experimental-design-data-analysis-and-inferential-statistics" rel="self"/>
+    <link href="http://www.zotero.org/styles//associacao-brasileira-de-normas-tecnicas" rel="template"/>
+    <link href="https://bjedis.org/author-1" rel="documentation"/>
+    <author>
+      <name>Fernando Gomes</name>
+      <email>fgsj@ufrj.br</email>
+    </author>
+    <category citation-format="numeric"/>
+    <summary>Style based on ABNT numeric, V1.1</summary>
+    <updated>2021-02-09T15:04:30+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="no date">[no date]</term>
+      <term name="in">in</term>
+      <term name="online">online</term>
+      <term name="accessed">accessed</term>
+      <term name="retrieved">Available</term>
+      <term name="from">from</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" name-as-sort-order="all">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
+      </name>
+      <label prefix=" (" form="short" suffix=".)"/>
+    </names>
+  </macro>
+  <macro name="translator">
+    <names variable="translator">
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
```

Comment:
all these given name name-parts don't do anything (you only need to use `name-part` when you apply additional casing or formatting). Please removes

Reply:
Thank you! They were removed.